.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Upgrade pip
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
I guess we don't need python setup if we are just building this inside a docker container. Though maybe it makes sense if the wheel is being installed?

Reply:
>Though maybe it makes sense if the wheel is being installed?

Yep, that was the motivation for it.